stock-tracker/src/features/chart/component/chart.tsx

```diff
@@ -18,24 +18,26 @@ import { ChartButton } from "./chartButton";
 import "./chart.css";
 import moment from "moment";
 import { AppState } from "../../../store/rootReducer";
+import { ChartTimes } from "../redux/actions";
 
 export const Chart: FC = () => {
   const dispatch = useDispatch();
   const { chartData, loading, error } = useSelector(
     (state: AppState) => state.chart
   );
-  const [current, setCurrent] = useState("1Y");
-
-  const onClick: React.MouseEventHandler<HTMLButtonElement> = e => {
-    dispatch(setChartTimeAction(e.currentTarget.value));
-    setCurrent(e.currentTarget.value);
-  };
+  const [current, setCurrent] = useState<ChartTimes>("1Y");
+  const chartRanges: ChartTimes[] = ["1D", "5D", "1M", "1Y", "5Y", "MAX"];
```

Comment:
This can be pulled out as a constant at the top of this file, outside of the component.